HomeController.java

```diff
@@ -32,6 +32,10 @@ public ResponseDto getPolitischeInformation(@RequestParam String mostSpeechesFor
 
             Map<String, Integer> mapPoliticians = new HashMap<>();
             String maxSpeechesForYearPolitik = null;
+            System.out.println("just testing");
+                        System.out.println("just testing");
+           while()

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;(&#34;, Expression expected after this token